modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -149,10 +149,40 @@ A GraphQL API schema designed with Apollo GraphQL specific directives and struct
 To create a GraphQL API:
 
 . In Exchange, select *Publish new asset*.
-. Enter the asset portal name.
-. From the drop-down list, select the asset type *GraphQL API* .
+. Enter a name for the asset.
+. From the drop-down list, select the asset type *GraphQL API*.
 . Select which GraphQL API schema to upload, GraphQL API schema or GraphQL Apollo API schema.
-. To select the file to upload, click *Choose the file* .
+. To select the file to upload, click *Choose file*.
```

Comment:
For these two that start with "To select", I sort of want to flip to have the imperative start the step.
Click ... to ...

Reply:
Hi, those steps are optional. See now that they should start with "If". I will fix this.

Reply:
I will fix this globally for all instances of optional steps.